.env.example

```
@@ -14,4 +14,9 @@ SERVER_PORT=8080
 SWAGGER_ENABLED=true
 
 CORS_ALLOWED_ORIGINS=http://localhost:3000
-CORS_ALLOWED_METHODS=GET,PUT,POST,PATCH,DELETE,OPTIONS
\ No newline at end of file
+CORS_ALLOWED_METHODS=GET,PUT,POST,PATCH,DELETE,OPTIONS
+
+MAIL_HOST=
+MAIL_PORT=
+MAIL_USERNAME=
+MAIL_PASSWORD=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)